ci-operator/step-registry/etcd-encryption/etcd-encryption-commands.sh

```diff
@@ -3,6 +3,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy"
+        # cat "${SHARED_DIR}/proxy-conf.sh"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "no proxy setting."
+    fi
+}
+set_proxy
```

Comment:
No need a function here, just make it simple.
```suggestion
[ -f "${SHARED_DIR}/proxy-conf.sh" ] && source "${SHARED_DIR}/proxy-conf.sh"
```